homeassistant/components/meteo_france/weather.py

```diff
@@ -83,8 +84,13 @@ def attribution(self):
     @property
     def forecast(self):
         """Return the forecast."""
-        reftime = datetime.now().replace(hour=12, minute=00)
+        reftime = dt_util.as_utc(
+            dt_util.now(dt_util.get_time_zone("Europe/Paris")).replace(
```

Comment:
Why would we use the Paris timezone and not the users timezone ?

Reply:
Your question has triggered a lot of brain activity. At first tough I was thinking the french weather service was providing forecast using Paris timezone.
After investigating weather forecast are done using the local area timezone.
Now I'm sure if I want correct timezone in the forecast it have to be manage on the package side. In the mean time I will us the user timezone assuming users are interesting mainly for forecast in their zone. Impact is low as currently the component support only day by day forecast.